### PR TITLE
Added HTML and CSS

### PR DESCRIPTION
**Pull request by ntazelo**
**Tribute page**

On this tribute page, I added :

- An HTML file

- A CSS file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+             Hollywood (and made it actually). He has like a million movies and he is
+             the most followed actor in Instagram.
+        </p>
+       <p>
+           If you want to know more about The Rock just go <a href="https://fr.wikipedia.org/wiki/Dwayne_Johnson"  target="_blank" id="tribute-link">here</a>
+        </p>
+    </main>
+
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+</body>
```
Comment (ntazelo) [inline index.html:32]: Please removed unnecessary blank spaces. The use of unnecessary blank spaces is not following the HTML & CSS best practices

File: style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#img-div{
+    text-align: center;
+    background: #eee;
+}
```
Comment (ntazelo) [inline style.css:4]: please fix the linter errors by using the correct indentation in your CSS file

File: index.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+            <p id="img-caption">The Rock posing intimidately.</p>
+         </div>
+
+         <p id="tribute-info">
+             The Rock is a former WWE fighter that really wanted to make it into
+             Hollywood (and made it actually). He has like a million movies and he is
+             the most followed actor in Instagram.
+        </p>
+       <p>
+           If you want to know more about The Rock just go <a href="https://fr.wikipedia.org/wiki/Dwayne_Johnson"  target="_blank" id="tribute-link">here</a>
```
Comment (ntazelo) [inline index.html:27]: please fix the linter error relate to the link with attribute target="__blank" by adding rel="noreferrer" rel="noopener"
attributes without those attribute your site may encounter security issues and bad performance.

**Review by ntazelo — COMMENTED** [review]
# Changes required ♻️  

Please check my comments below and make all required changes.

 Submit for a re-review when you're done.

Thanks 😁

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**